### ISSUE TITLE
Deploy the tank logo for the website, official PR channels and the installer.

### ISSUE THREAD

**Issue by GraionDilach** [OpenRA/OpenRA#14272]
When the developer intention of the tank logo was announced, a loud portion of the playerbase replied with the fact they prefer the hammer and sickle logo because "it's RA". With the removal of the mod switcher and the inclusion of the separate mod executables, the complaints of this portion of the playerbase became invalid.

This would be another step towards silencing the the vocal complaints on PPM with demanding a name change, ignoring the strength of the brand and the history of the project and another step to propagate OpenRA-as-a-game-engine.

**Comment by netnazgul** [comment]
Didn't see the logo that is being promoted here (if it exists then it's better to display it here, otherwise raise the discussion on its drawing), but as the executables are separate now and each of primary mods has the dedicated icon, I don't see any obstacles to have a dedicated logo for the project which is [yet] bigger than any of the mods it contains.

**Comment by GraionDilach** [comment]
There wasn't a ticket on GH, but it was mentioned a lot of times on the IRC and received it's own newspost.

http://www.openra.net/news/new-project-logo/

**Comment by netnazgul** [comment]
I do see the reason behind the comments about this pic being too "busy" with details. Probably it just looks too generic with so many details on it. However it doesn't that bad and still is a decent feature image for the project. 

As for the icon - currently there is even no need for the 32x32 icon because mods are separated and have their dedicated icons.

**Comment by reaperrr** [comment]
> "OpenRTS"

Already taken, afaik (otherwise that would have been the obvious choice, if we ever decided to rename the project).

**Comment by mabulsoud** [comment]
netnazgul's point about style mixture is on point. It may be worth it to see what it would look like if the tank is also pixelated.

It might be worth it to start from scratch as well with a more minimal design approach that would communicate more to devs than gamers, as this would be the goal at this point after the mod separation at this point.

I am not in the position to make decisions on the names, but I must say I think there is something to the idea of moving the engine away from "OpenRA" and using that for the mods themselves as suggested here https://ppmforums.com/viewtopic.php?t=44236 .

**Comment by GraionDilach** [comment]
>It might be worth it to start from scratch as well with a more minimal design approach that would communicate more to devs than gamers, as this would be the goal at this point after the mod separation at this point.

What modders/developers should see that this is an RTS engine, providing old 2(.5D) engine POV with good capabilities. The tank having a detailed look (implying modern capabilities) while still being fullremap with the pixellish mouse cursor (remiscent of '90s games) and the health bar achieves all of these aims.

The style mixture is IMO exactly on the point with the style mixture of the currently-shipped mods (assets from 1995 used for gameplay channeling these days).

The PPM crowd have had multiple attacks based on their claimed opinion that OpenRA isn't "worthy" of calling itself "RA" because "it's too far away from the Westwood vision". That suggestion is just another attempt of executing their malicious intent. Throwing away the brand should never happen, considering that the project has a close decade of history now. Unreal Engine is still named after the first game it's used within instead of an abstract name after Epic Games decided to sell it either - yet UE4 isn't strictly FPS/TPS engine these days (I'm looking at you, Tekken 7). Same applies to CryEngine as well.

**Comment by mabulsoud** [comment]
Let me clarify what my intention is regarding the name change then:

RA mod ->OpenRA
TD mod ->OpenTD
D2k mod ->OpenD2k
OpenRA Engine -> (new brand name)

That would mean that the OpenRA brand isn't completely lost, but the implication of RA is limited to the RA mod. What you brought up with the example of the unreal engine is perfectly valid and could also apply here as well, this is merely a suggestion since "rebranding" is being brought up again.

As far as the logo is concerned, perhaps I should have stated that I'm perfectly happy with the way the logo turned out for what was it's intention at the time, which are exactly what you had pointed out, I'm merely bringing up the fact that perhaps the intent this time around could be more forward looking, and not necessarily tied to the 90's game revival theme.

That being said, I still would not really know where to start on an alternative logo, I'm merely stating that I would work on the logo again if it's needed, which I couldn't say I was the last few months.

**Comment by GraionDilach** [comment]
OpenTD is can be easily mixed with OpenTTD and there was a defunct project called OpenCnC already. The mods were always explicitly called "RA mod/TD mod/etc" and such in official discussions exactly to prevent any implication of the exact name setup you're just suggesting.

Thing is that post-90's games are about 3D environments and OpenRA is completely cell-based (which won't be removed), and I believe that the mouse channels this fact well.

**Comment by mabulsoud** [comment]
Fair enough, I'm not going to argue too hard to create more work for myself, in the end it's the devs' decision.

**Comment by anjew175** [comment]
It might not be appreciated if i place this here but this was the 2 sleipnir discussion and the PPM discussion.
http://www.sleipnirstuff.com/forum/viewtopic.php?f=82&t=19917
http://www.sleipnirstuff.com/forum/viewtopic.php?t=19935
https://ppmforums.com/viewtopic.php?t=43050

**Comment by Mailaender** [comment]
During the times where I was answering our fan posts at Facebook and ModDB (a long time ago) I was told that the hammer & sickle symbol has a negative connotation in Eastern European countries. In fact it is a forbidden symbol in those countries just like the Swastika is in Germany. This is not my personal opinion. I just want to give some reason to this very emotional debate.

**Comment by netnazgul** [comment]
@Mailaender I think the main debate is not about leaving the current logo as it is (because if anything - it doesn't do all the things mentioned in the OP and it's already a logo for original game, which doesn't correlate with the engine), but about new proposed logo being not refined enough as a "logo"

**Comment by Mailaender** [comment]
We could use it for https://github.com/OpenRA/OpenRAModSDK/tree/master/packaging/artwork
